### PR TITLE
Improve docstring

### PR DESCRIPTION
**Pull request by chezou**
with minor fixes for some flake8 warning/errors.

### CODE REVIEW

File: tdclient/bulk_import_api.py [hunk 1]
```diff
@@ -11,9 +11,10 @@
 
 
 class BulkImportAPI:
-    ####
-    ## Bulk import API
-    ##
+    """Enable bulk importing of data to the targeted database and tableAccess.
```
Comment (takuti) [inline tdclient/bulk_import_api.py]: typo `s/Access//`

File: tdclient/result_api.py [hunk 2]
```diff
@@ -4,16 +4,17 @@
...
+    """
 
     def list_result(self):
         """Get the list of all the available authentications.
 
         Returns:
-            [(str, str, None)]: The list of tuple of name, Result output url, and None
-                (organization name).
+            [(str, str, None)]: The list of tuple of name, Result output url, and
+                 organization name (always `None`).
```
Comment (takuti) [inline tdclient/result_api.py]: Write the reason of why `None` is needed as: *for API copatibility*

File: tdclient/job_model.py [hunk 3]
```diff
@@ -210,7 +210,7 @@ def wait(self, timeout=None, wait_interval=5, wait_callback=None, callback=None)
         """Sleep until the job has been finished
 
         Args:
-            timeout (int): Timeout in seconds. No timeout by default.
+            timeout (int, optional): Timeout in seconds. No timeout by default.
             wait_interval (int): wait interval in second. Default 5 seconds.
             wait_callback (callable): A callable to be called on every tick of wait interval.
```
Comment (takuti) [inline tdclient/job_model.py]: Are they optional as well?

File: tdclient/connector_api.py [hunk 4]
```diff
@@ -78,12 +83,14 @@ def connector_list(self):
...
         Args:
-          name (str): name of the connector job
-          database (str): name of the database to perform connector job
-          table (str): name of the table to perform connector job
-          job (dict): :class:`dict` representation of `load.yml`
+            name (str): name of the connector job
+            database (str): name of the database to perform connector job
+            table (str): name of the table to perform connector job
+            job (dict): :class:`dict` representation of `load.yml`
+            params (dict, optional): Extra parameters
```
Comment (takuti) [inline tdclient/connector_api.py]: Is it possible to list what parameters are available?

Comment (chezou) [reply tdclient/connector_api.py]: addedd 5913169

File: tdclient/job_model.py (hunk 3 above)
Comment (chezou) [reply tdclient/job_model.py]: There are. but found another major issue, created another PR https://github.com/treasure-data/td-client-python/pull/74

**Pull request merged**